lib/prometheus/client/counter.rb

```diff
@@ -11,14 +11,12 @@ def type
       end
 
       def increment(labels = {}, by = 1)
+        fail ArgumentError, 'increment must be a postive integer' if by < 1
```

Comment:
Hm. Right, actually it's about floats (damn untyped languaged). I don't see too much sense in increasing by 0, but fair enough given it's the same as not increasing at all.